reana_server/reana_admin/options.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of REANA.
+# Copyright (C) 2022 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+"""Common options for the REANA administrator command line tool."""
+
+import sys
+import click
+import functools
+
+from reana_db.utils import _get_workflow_by_uuid
+
+from reana_server.utils import _get_user_by_criteria, is_uuid_v4
+
+
+def add_user_options(func):
```

Comment:
My idea is that we can use `add_user_options` and `add_workflow_option` everywhere, so that we can avoid repeating the logic to validate inputs and to query the database to get the corresponding user/workflow. What do you think?

Reply:
I think it's a nice idea. But we should make sure it covers all the use cases, for example it could be used in `retention-rules-extend` command, but there the flag is set to be `required=True`. So it would be nice to also be able to pass additional arguments here. We could create an issue for later to extend it and to use these utilities in other cli commands